docs/context/ref/Email.md

```diff
@@ -0,0 +1,60 @@
+---
+id: Email
+sidebar_label: Email
+title: Email
+hide_title: true
+---
+# `Email`
+
+A collection of information to be used to initiate an email with a Contact or ContactList
+
+## Type
+
+`fdc3.email`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/email.schema.json
+
+## Details
+
+| Property          | Type                                  | Required | Example Value       |
+|-------------------|---------------------------------------|----------|---------------------|
+| `type`            | string                                | Yes      | `'fdc3.email'` |
+| `recipients`      | fdc3.contact or fdc3.contactList      | Yes      | { type: "fdc3.contact", name: "John Doe", id: { "email": "john@sample.com"}} |
+| `subject`         | string                                | No       | `'The information you requested'`            |
+| `textBody`        | string                                | No       | `'Blah, blah, bah`         |
+| `richTextBody`    | fdc3.richTextBlock                    | No       | See https://github.com/finos/FDC3/issues/575        |
```

Comment:
Needs updating if that issue/PR is merged, or removing if not...

Reply:
I'm not sure what the request is here

Reply:
@pauldyson It was mostly to remind me to make sure this is sorted before we merge.

This PR refers to issue #575 - we, need to resolve that reference before we release if #575 has a PR that gets merged (which has not been raised yet  - correct me if I'm wrong on that @bertrand-s) or remove this/downgrade to a note that it might be added in future.